src/applications/claims-status/components/ClaimsListItem.jsx

```diff
@@ -47,6 +50,14 @@ CommunicationsItem.propTypes = {
   icon: PropTypes.string.isRequired,
 };
 
+const clickHandler = href => {
```

Comment:
Btw, link clicks are already automatically tracked by google analytics, I would imagine this and the other link clicks are already being tracked in GA